ReleaseChecklist.md

```diff
@@ -32,6 +32,10 @@
 ### Homebrew and MacOS
  - [ ] Update the version and the hash (``sha256sum solidity_x.x.x.tar.gz``) in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb
  - [ ] Take the binary from the ``b_osx`` run of the released commit in circle-ci and add it to the release page as ``solc-macos``.
+ - [ ] bump new homebrew formula
+```
```

Comment:
Probably better to use placeholders for the version number and the hash.

Please also mention what has to be done to do this manualy.